### ISSUE TITLE
PBSV: Fails parsing BED files with a header

### ISSUE THREAD

**Issue by paudano** [PacificBiosciences/pbbioconda#422]
Error:
`>|> 20210720 20:38:49.156 -|- FATAL -|- Run at ../subprojects/pbcopper/src/cli2/CLI.cpp:139 -|- 0x2aaaaaae4300|| -|- pbsv discover ERROR: Could not parse repeat bed file, line 0`

**Comment by zeeev** [comment]
Hi @paudano, 

Hope you're doing well. We'll take a look at this soon and get back to you.

**Comment by zeeev** [comment]
@paudano, if you remove the header does it work for you?

**Comment by paudano** [comment]
Yes, I was able to run PBSV using the headerless version. Valid BED files can have headers, so I'm letting you know (low priority, not blocking me).

Thanks Zev!
